### PR TITLE
Harmonize wordings of notifications pan

### LINKED ISSUE #23479: Harmonize the wording for notifications
#### Is your feature request related to a problem?

The wording for notifications (new orders, messages, customers) is different depending on the page the user is on. 

#### Describe the solution you'd like

I would like to harmonize the wording following @prestascott's [idea](https://github.com/PrestaShop/PrestaShop/pull/22519#issuecomment-788020413):

Delete 'New' and 'Latest' to keep only 'Orders', 'Customers' and 'Messages' in _Admin.Navigation.Header_

ORDERS:
Keep: 'No new order for now :('
'Have you checked your abandoned carts?'
'Your next order could be hiding there!' in _Admin.Navigation.Notification_
See https://github.com/PrestaShop/PrestaShop/pull/22519#issuecomment-788020413

CUSTOMERS:
'No new customer for now :('
'Are you active on social media these days?' in _Admin.Navigation.Notification_

MESSAGES:
'No new message for now.'
'Seems like all your customers are happy :)' in _Admin.Navigation.Notification_

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Wordings were always different on notification window
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #23479.
| How to test?      | Go on the BO on migrated AND legacy pages, see wordings

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23502)

### CODE REVIEW

File: classes/controller/AdminController.php [hunk 1]
```diff
@@ -2054,8 +2054,6 @@ private function getNotificationTip($type)
     {
         $tips = [
             'order' => [
-                $this->trans('Did you check your conversion rate lately?', [], 'Admin.Navigation.Notification'),
```
Comment (sowbiba) [inline classes/controller/AdminController.php:2057]: I don't get why you remove these translations
What was their usage ?

**Review by matthieu-rolland — COMMENTED** [review]
Same question as @sowbiba about the removed translations, otherwise LGTM 👍

File: classes/controller/AdminController.php (hunk 1 above)
Comment (NeOMakinG) [reply classes/controller/AdminController.php:2057]: They were randomly used on page load, if you feel like you can remove the "random thing"...

Comment (PierreRambaud) [reply classes/controller/AdminController.php:2057]: Pages are not migrated, afaik these strings are not used anymore :thinking:

Comment (NeOMakinG) [reply classes/controller/AdminController.php:2057]: They are 100% used as I tested this, in the notification window, but maybe I had luck and launched on the same string on 4-5 refresh of the page?

**Review by matthieu-rolland — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by aDamelincourt** [comment]
✔️
All wordings are now consistent between legacy and Symphony screens

OK_PR23502_Screenshot 2021-03-11_16-40-36-861
OK_PR23502#2_Screenshot 2021-03-11_16-41-18-209
OK_PR23502#3_Screenshot 2021-03-11_16-42-00-543

Thanks @NeOMakinG

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @aDamelincourt

**Pull request merged**